src/evaluator.ts

```diff
@@ -534,7 +534,10 @@ class Evaluator {
     const table: EvaluateTable = {
       head: {
         prompts,
-        vars: Array.from(varNames).sort(),
+        vars: [
+          ...Object.keys(testSuite.defaultTest?.vars || {}).sort(),
+          ...Array.from(varNames).sort(),
+        ],
```

Comment:
nit, we should consider sorting everything together or nothing at all if we want to preserve template order.

imagine things  are specified as 

```
default
 - Bdefault
 - Adefault
 
 vars: 
 - A 
 - B
```

Do we want 
```
- Adefault
- Bdefault 
- A
- B
```

or 

```
- A
- Adefault
- B
- Bdefault 
```

or something else